mashumaro/core/meta/code/builder.py

```diff
@@ -818,7 +818,11 @@ def _add_pack_method_lines(self, method_name: str) -> None:
             packers = {}
             aliases = {}
             fields_could_be_none = set()
-            for fname, ftype in field_types.items():
+            fnames_and_types = list(field_types.items())
```

Comment:
We don't need a list if `sort_keys` is not set:
```python
fnames_and_types = field_types.items()
if self.get_config().sort_keys:
    fnames_and_ftypes = sorted(
        fnames_and_ftypes, key=lambda x: x[0]
    )  # type: ignore
```

Reply:
@Fatal1ty thanks. not a big fan of untyped code, so I used a typed version of your suggestion